modules/cluster-logging-upgrading-logging.adoc

```diff
@@ -5,98 +5,135 @@
 [id="cluster-logging-updating-logging_{context}"]
 = Updating cluster logging
 
-After updating the {product-title} cluster, you can update cluster logging from 4.2 to 4.3 by updating the subscription for the Elasticsearch Operator and the Cluster Logging Operator.
+[role="_abstract"]
+After updating the {product-title} cluster, you can update cluster logging from 4.4 to 4.6 by changing the subscription for the Elasticsearch Operator and the Cluster Logging Operator.
+
+When you update:
+
+* You must update the Elasticsearch Operator before updating the Cluster Logging Operator.
+* You must update both the Elasticsearch Operator and the Cluster Logging Operator.
++
+Kibana is unusable when the Elasticsearch Operator has been updated but the Cluster Logging Operator has not been updated.
++
+If you update the Cluster Logging Operator before the Elasticsearch Operator, Kibana does not update and the Kibana custom resource (CR) is not created. To work around this problem, delete the Cluster Logging Operator pod. When the Cluster Logging Operator pod redeploys, the Kibana CR is created.
 
 [IMPORTANT]
 ====
-Changes introduced by the new log forward feature modified the support for *out_forward* starting with the {product-title} 4.3 release. In {product-title} 4.3, you create a ConfigMap to configure *out_forward*. Any updates to the `secure-forward.conf` section of the Fluentd ConfigMap are removed.
-
-If you use the *out_forward* plug-in, before updating, you can copy your current `secure-forward.conf` section from the Fluentd ConfigMap and use the copied data when you create the `secure-forward` ConfigMap. 
+If your cluster logging version is prior to 4.5, you must upgrade cluster logging to 4.5 before updating to 4.6.
```

Comment:
That is headache. There isn't 4.5 packages in 4.6 cluster.  Prior 4.5, OLM use operator-source to provide packages, this operator-source can provide operator 4.1,4,2,4,3,4,4,4,5 packages.   But from 4.6, OLM use catalogsoure to provide packages, you can only see the v4.6 packages in 4.6 cluster.   so you must complete the 4.x ->4.5 upgrade in the 4.5 cluster and prior.

Reply:
@vikram-redhat   In fact, we can upgrade from 4.4 to 4.6 directly with some workaround. I like the option b). as that needn't restart elasticsearch cluster manually. 
a. Upgrade elasticsearch-operator first.
http://pastebin.test.redhat.com/946177
b. Upgrade cluster-logging-operator first.
http://pastebin.test.redhat.com/946284

Reply:
@ewolinetz can you give this a look please?
iirc we could still go directly to 4.6 from 4.4

Reply:
technically our skiprange and code should it, i'll leave it to @anpingli on whether or not this has been tested and verified though (it seems like it has been)

Reply:
@anpingli Please advise as to what the documentation should say so that we can resolve this comment. Thanks.

Reply:
@ewolinetz  You can upgrade from v4.4 to v4.6. For sure,  you will hit some error. See the pasterbin above.

Reply:
@jonquilwilliams  @ewolinetz   I suggest updating as below. WDYT?
1) list issue and workaround before upgrade. and ask the customer to under standard this issue.
2) Upgrade to the latest v4.4 before upgrade to v4.5.
3) Upgrade to the latest v4.5 before upgrade to v4.6.
4) We suggest upgrade version by version.  If the OCP had been upgraded to v4.6, Logging still in v4.4.  Now  no way to upgrade version by version.  You can upgrade Logging from v4.4 to v4.6 directly. But you may hit the known the issues. Please follow the issue/workaround to fix it.
2)

Reply:
@ewolinetz Please comment on Anping's proposal so that we can resolve this PR. It sounds like Anping wants your agreement, and I need to know you're both on the same page before we can update the documentation. Thank you.

CC: @vikram-redhat  @RiRa12621  @wgordon17

Reply:
the suggestion sgtm